Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Select.hs

```diff
@@ -129,6 +129,8 @@ select personId estimateId req@DEstimateSelectReq {..} = do
     QPFS.updateStatus searchRequest.riderId DPFS.WAITING_FOR_DRIVER_OFFERS {estimateId = estimateId, validTill = searchRequest.validTill}
     QEstimate.updateStatus estimateId $ Just DEstimate.DRIVER_QUOTE_REQUESTED
     QEstimate.updateAutoAssign estimateId autoAssignEnabled (fromMaybe False autoAssignEnabledV2)
+    when (isJust req.customerExtraFee) $ do
+      QSearchRequest.updateCustomerExtraFee searchRequest.id req.customerExtraFee
```

Comment:
As i said to someone, i strongly against updating old entities, in cases when it doesn't make any sense from flow perspective. But fine, just one more thing to fix when we merge my declined PR with `BPP.SearchRequest` at `search` flow step...